docs/DeveloperGuide.md

```diff
@@ -158,42 +158,52 @@ This section describes some noteworthy details on how certain features are imple
 
 #### Proposed Implementation
 
-The proposed undo/redo mechanism is facilitated by `VersionedDengueHotspotTracker`. It extends `DengueHotspotTracker` with an undo/redo history, stored internally as an `dengueHotspotTrackerStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+The proposed undo/redo mechanism is facilitated by `TemporaryMemory`. It extends `SpecialisedStackForMemory`, which implements `StackWithStorage`.
+`TemporaryMemory` only stores the 10 most recent actions performed by the user, when the app is open. This means that when the app is closed and open again, the user will not be able to perform an undo or redo.
+`TemporaryMemory` can be viewed as a stack which supports additional operations.
 
-* `VersionedDengueHotspotTracker#commit()` — Saves the current Dengue Hotspot Tracker state in its history.
-* `VersionedDengueHotspotTracker#undo()` — Restores the previous Dengue Hotspot Tracker state from its history.
-* `VersionedDengueHotspotTracker#redo()` — Restores a previously undone Dengue Hotspot Tracker state from its history.
+1. `TemporaryMemory` only contains 10 saved iterations of the file. Therefore, older iterations are deleted.
+2. `TemporaryMemory` supports the redo command, and therefore, after performing an undo, more recent iterations of the file are still stored in a temporary storage.
```

Comment:
Might be better to refer to "temporary storage" by another name, since it might be confused with TemporaryMemory / the storage that holds past iterations, rather than future ones that can be redone!

Reply:
```suggestion
2. `TemporaryMemory` supports the redo command, and therefore, after performing an undo, more recent iterations of the file are still stored in an auxiliary storage component.
```